src/Mondu/Mondu/Gateway.php

```diff
@@ -126,4 +128,15 @@ public function process_payment($order_id) {
       'redirect' => $this->get_return_url($order)
     );
   }
+
+  /**
+   * Check if Mondu has its credentials validated.
+   *
+   * @return string
+   */
+  private function is_enabled() {
```

Comment:
suggestion (non-blocking): Maybe we can create a MonduGateway class that extends WC_Payment_Gateway. And use all repeated code in ( Gateway/GatewayDirectDebit/GatewayInstallment) there ?

So instead of Gateway extends WC_Payment_Gateway we will gave Gateway extends MonduGateway...